docs/Reference/Errors.md

```diff
@@ -290,6 +290,18 @@ The logger accepts either a `'stream'` or a `'file'` as the destination.
 The logger should have all these methods: `'info'`, `'error'`,
 `'debug'`, `'fatal'`, `'warn'`, `'trace'`, `'child'`.
 
+#### FST_ERR_LOG_INVALID_LOGGER_INSTANCE
+<a id="FST_ERR_LOG_INVALID_LOGGER_INSTANCE"></a>
+
+The loggerInstance only accepts a logger instance, not configuration object. 
+See ([#4624](https://github.com/fastify/fastify/issues/4624))
```

Comment:
```suggestion
```

We should have self contained docs. I dont see any additional information in the issue. :)